--- src/Info.plist.src	2008-06-26 20:07:05.000000000 -0700
+++ src/Info.plist.src	2008-07-29 16:09:52.000000000 -0700
@@ -1,13 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
+	<key>CFBundleExecutable</key>
+	<string>ifd-ccid.bundle</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.apple.smartcardccid</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>CCIDCLASSDRIVER</string>
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.3.8</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
@@ -28,7 +28,7 @@
 	<string>0x00000001</string>
 
 	<key>ifdLogLevel</key>
-	<string>0x0003</string>
+	<string>0x0001</string>
 
 	<!-- Possible values for ifdLogLevel
 	1: CRITICAL  important error messages
