diff --git MacOSX/configure MacOSX/configure
index 8b42e38..b3fc826 100755
--- MacOSX/configure
+++ MacOSX/configure
@@ -39,6 +39,9 @@ fi
 
 ./MacOSX/convert_reader_h.pl MacOSX/reader.h.in > MacOSX/reader.h
 
+if [[ "$*" =~ "--no-configure" ]]; then
+	exit 0
+fi
 # we define PKG_CONFIG so that pkg-config is not used and PCSC_CFLAGS
 # and PCSC_LIBS are used instead
 PKG_CONFIG="foobar"
@@ -47,9 +50,16 @@ PKG_CONFIG="foobar"
 PCSC_CFLAGS="-I$(pwd)/MacOSX"
 PCSC_LIBS="-framework PCSC"
 
-# use libusb from OpenSC (universal binary)
-LIBUSB_CFLAGS="-I/Library/OpenSC/include"
-LIBUSB_LIBS="-L/Library/OpenSC/lib -lusb"
+# If --disable-opensc given, then do not alter the LIBUSB environment
+if [[ ! ("$*" =~ "--disable-opensc") ]]; then
+	# use libusb from OpenSC (universal binary)
+	LIBUSB_CFLAGS="-I/Library/OpenSC/include"
+	LIBUSB_LIBS="-L/Library/OpenSC/lib -lusb"
+
+	CONFIGURE_ARGS="$CONFIGURE_ARGS \
+		LIBUSB_CFLAGS=\"$LIBUSB_CFLAGS\" \
+		LIBUSB_LIBS=\"$LIBUSB_LIBS\""
+fi
 
 # RESPONSECODE is already defined by PCSC/wintypes.h
 # define needed here to compile examples/scardcontrol.c since config.h is
@@ -59,7 +69,7 @@ CFLAGS="$CFLAGS -DRESPONSECODE_DEFINED_IN_WINTYPES_H"
 # Build a Universal Binary
 CFLAGS="$CFLAGS -isysroot /Developer/SDKs/MacOSX10.5.sdk -arch i386 -arch ppc"
 LDFLAGS="-arch i386 -arch ppc"
-CONFIGURE_ARGS="--disable-dependency-tracking"
+CONFIGURE_ARGS="$CONFIGURE_ARGS --disable-dependency-tracking"
 
 # do not use pcscd debug feature
 CONFIGURE_ARGS="$CONFIGURE_ARGS --disable-pcsclite"
@@ -70,8 +80,6 @@ set -x
 	CFLAGS="$CFLAGS" \
 	PCSC_CFLAGS="$PCSC_CFLAGS" \
 	PCSC_LIBS="$PCSC_LIBS" \
-	LIBUSB_CFLAGS="$LIBUSB_CFLAGS" \
-	LIBUSB_LIBS="$LIBUSB_LIBS" \
 	LDFLAGS="$LDFLAGS" \
 	--enable-usbdropdir=/usr/libexec/SmartCardServices/drivers \
 	$CONFIGURE_ARGS \
